plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/HttpThriftMetastoreClientFactory.java

```diff
@@ -56,7 +56,7 @@ public class HttpThriftMetastoreClientFactory
     private final Map<String, String> additionalHeaders;
     private final OpenTelemetry openTelemetry;
 
-    private final AtomicInteger chosenGetTableMetaAlternative = new AtomicInteger(Integer.MAX_VALUE);
+    private final AtomicInteger chosenGetTableMetaAlternative = new AtomicInteger(1);
```

Comment:
this looks like pre-choosing an alternative?
should `chosenGetTableMetaAlternative` be removed?

Reply:
applied